torch/onnx/symbolic_helper.py

```diff
@@ -131,6 +131,12 @@ def _parse_arg(value, desc, arg_name=None, node_name=None):
         )
 
 
+def _node_get(node: _C.Node, key: str):
```

Comment:
Should we mention `attr` in the function name?

Reply:
I think we can opt for simplicity in this case, since I don't foresee getting other things from the node. If it becomes necessary that we distinguish between different get methods, we can rename the function or design a more elaborate mechanism for working with nodes.

Reply:
But this also raises a good point on how we can supply a set of helpers to users who what to create their own symbolics. if one needs to obtain a value in a node, all they can do now is replicate this function and use that. Since we are usually calling private helpers, users may follow our usage and inevitably use those private helpers too

Reply:
sgtm